display_information/update-graphics/README.md

```diff
@@ -11,8 +11,8 @@ A field worker may want to update a graphic's properties to adjust which real-wo
 ## How to use the sample
 
 Update Graphic's Location:
-  - click on a graphic, then click the update location button
-  - now click a new location on the map to move graphic
+  - click on a graphic, then drag the graphic to a new location
```

Comment:
It might be really good to have the ability to directly drag graphics without clicking them first. It might complicate the code a bit, but worth looking into.

I've notice that if I try to drag without the initial click to select the graphics first, there's a `NullPointerException`, meaning we need to check on drag whether there's a graphics selected.